src/deploy/osp_deployer/sah.py

```diff
@@ -398,6 +404,75 @@ def delete_director_vm(self):
             self.run("virsh undefine director")
             time.sleep(20)
 
+    def create_powerflexgw_vm(self):
+        powerflexgw_conf = "/root/powerflex-gw.cfg"
+        self.run("rm " + powerflexgw_conf + " -f")
+        conf = ("rootpassword " + self.settings.powerflexgw_vm.root_password,
+                "timezone " + self.settings.time_zone,
+                "smuser " + self.settings.subscription_manager_user,
+                "smpassword " + self.settings.subscription_manager_password,
+                "smpool " + self.settings.subscription_manager_pool_vm_rhel,
+                "hostname " + self.settings.powerflexgw_vm.hostname + "." +
+                self.settings.domain,
+                "gateway " + self.settings.public_api_gateway,
+                "nameserver " + self.settings.name_server,
+                "ntpserver " + self.settings.sah_node.provisioning_ip)
+        if self.settings.use_satellite is True:
+            conf = conf + ("satellite_ip " + self.settings.satellite_ip,)
+            conf = conf + ("satellite_hostname " +
+                           self.settings.satellite_hostname,)
+            conf = conf + ("satellite_org " + self.settings.satellite_org,)
+            conf = conf + ("satellite_activation_key " +
+                           self.settings.satellite_activation_key,)
+
+        conf = conf + ("# Iface     IP" +
+                       "               NETMASK" +
+                       "              MTU",)
+        conf = conf + ("enp1s0        " +
+                       self.settings.powerflexgw_vm.public_api_ip +
+                       "    " + self.settings.public_api_netmask +
+                       "     " + self.settings.public_api_network_mtu,)
+        conf = conf + ("enp2s0        " +
+                       self.settings.powerflexgw_vm.storage_ip +
+                       "    " + self.settings.storage_netmask +
+                       "     " + self.settings.storage_network_mtu,)
+
+        for line in conf:
+            self.run("echo '" +
+                     line +
+                     "' >> " +
+                     powerflexgw_conf)
+        remote_file = "sh /root/deploy-powerflexgw-vm.sh " + \
+                      powerflexgw_conf + " " + \
+                      "/store/data/iso/RHEL8.iso"
```

Comment:
RHEl8 iso locatio, is it hardcoded? I thought the location comes from the ini file. I could be wrong

Reply:
Again, all this code was cloned from the director method and as you can see below, it's hardcoded.
        remote_file = "sh /root/deploy-director-vm.sh " + \
                      director_conf + " " + \
                      "/store/data/iso/RHEL8.iso"